test/utils/createClientRender.js

```diff
@@ -25,13 +41,43 @@ function clientRender(element, options = {}) {
 export function createClientRender(globalOptions = {}) {
   const { strict: globalStrict } = globalOptions;
 
+  const baseElement = document.createElement('div');
+  baseElement.className = 'rtl--baseElement';
+  document.body.appendChild(baseElement);
+
   return function configuredClientRender(element, options = {}) {
     const { strict = globalStrict, ...localOptions } = options;
 
-    return clientRender(element, { ...localOptions, strict });
+    return clientRender(element, { ...localOptions, baseElement, strict });
   };
 }
 
+const fireEvent = Object.assign(rtlFireEvent, {
+  // polyfill event.key for chrome 49 (supported in Material-UI v4)
```

Comment:
Would using `react-dom/test-utils` solve the issue?

Reply:
They don't create native events for you but dispatch a given one. I haven't looked for a package that creates "old events" ergonomically. I couldn't even find a way to create events in chrome 49 manually without react failing to identify the key.

It seems like `key` is the prioritized property to create the synthetic event from which makes this the simplest solution.

Reply:
So using `react-dom/test-utils` doesn't have an impact on how React polyfills the events.
What allowed the tests to pass previously in Chrome 49?

Reply:
It wasn't an actual event just something that matched it structurally for the important parts. It works most of the time since we're not interested in the full event object but it makes the test a bit more robust against implementation changes while they are more brittle towards browser implementation.

I would definitely rethink this if we would support more older browsers but since it's only chrome 49 and we will likely phase that out in v5 I'm comfortable with that.

It also raises awareness about behavior that would require user/dev action in certain browsers like enabling touch events in edge.